cypress.config.ts

```diff
@@ -22,6 +22,11 @@ export default defineConfig({
       on('task', {
         reset: resetStubs,
         ...auth,
+
+        stubSignIn: () => auth.stubSignIn([]),
+        stubSignInAsUserWithEditAuthority: () => auth.stubSignIn(['ROLE_EDUCATION_WORK_PLAN_EDITOR']),
+        stubSignInAsUserWithViewAuthority: () => auth.stubSignIn(['ROLE_EDUCATION_WORK_PLAN_VIEWER']),
```

Comment:
I imagine we'll use this stub and associated role once we get further into the dev, where they have stories for "read only" views of the data